x-pack/test/functional/page_objects/index_management_page.ts

```diff
@@ -16,6 +17,46 @@ export function IndexManagementPageProvider({ getService }: FtrProviderContext)
     async reloadIndicesButton() {
       return await testSubjects.find('reloadIndicesButton');
     },
+    async toggleRollupIndices() {
+      await testSubjects.click('checkboxToggles-rollupToggle');
+    },
+
+    async getIndexList() {
+      const indices = await testSubjects.findAll('indexTableRow');
+      return mapAsync(indices, async index => {
+        const indexNameLinkCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableIndexNameLink"]'
+        );
+        const indexHealthCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-health"]'
+        );
+        const indexStatusCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-status"]'
+        );
+        const indexPrimaryCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-primary"]'
+        );
+        const indexReplicasCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-replica"]'
+        );
+        const indexDocumentsCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-documents"]'
+        );
+        const indexSizeCell = await index.findByCssSelector(
+          '[data-test-subj="indexTableCell-size"]'
+        );
+
+        return {
+          indexName: await indexNameLinkCell.getVisibleText(),
+          indexHealth: await indexHealthCell.getVisibleText(),
+          indexStatus: await indexStatusCell.getVisibleText(),
+          indexPrimary: await indexPrimaryCell.getVisibleText(),
+          indexReplicas: await indexReplicasCell.getVisibleText(),
+          indexDocuments: await indexDocumentsCell.getVisibleText(),
+          indexSize: await indexSizeCell.getVisibleText(),
+        };
+      });
+    },
```

Comment:
Agree with @LeeDr , it is better to avoid multiple webDriver API calls via `mapAsync`. Cheerio will do it way faster with just 2 webDriver calls:
```
const table = await testSubjects.find(<table_element_id>);
const $ = await table.parseDomContent();
return $.findTestSubjects('indexTableRow').toArray().map(row =>  {
        return {
             indexName: $(row).findTestSubject('indexTableIndexNameLink').text(),
            ...
        };
});
```

```

Reply:
Interesting. I will change that. I didn't know that. I followed the convention in `rollup_page.js`. Did we not have that functionality when that was merged? Should I change that too?

Reply:
> Agree with @LeeDr , it is better to avoid multiple webDriver API calls via `mapAsync`. Cheerio will do it way faster with just 2 webDriver calls:
> 
> ```
> const table = await testSubjects.find(<table_element_id>);
> const $ = await table.parseDomContent();
> return $.findTestSubjects('indexTableRow').toArray().map(row =>  {
>         return {
>              indexName: $(row).findTestSubject('indexTableIndexNameLink').text(),
>             ...
>         };
> });
> ```

Is this documented somewhere? Do we have updated FTR docs?

Reply:
Fixed.